tests/drawer/test_draw.py

```diff
@@ -329,6 +331,51 @@ def func():
 
         assert drawing == expected_drawing
 
+    @pytest.mark.parametrize(
+        "op", [qml.GlobalPhase(0.1), qml.Identity(), qml.Snapshot(), qml.Barrier()]
+    )
+    @pytest.mark.parametrize("decimals", [None, 2])
+    def test_draw_all_wire_ops(self, op, decimals):
+        """Test that operators acting on all wires are drawn correctly"""
+
+        def func():
+            qml.X(0)
+            qml.X(1)
+            m = qml.measure(0)
+            qml.cond(m, qml.X)(0)
+            qml.apply(op)
+            return qml.expval(qml.Z(0))
+
+        # Stripping to remove trailing white-space because length of white-space at the
+        # end of the drawing depends on the length of each individual line
+        drawing = qml.draw(func, decimals=decimals)().strip()
+        label = op.label(decimals=decimals).replace("\n", "")
+        expected_drawing = (
+            f"0: ──X──┤↗├──X──{label}─┤  <Z>\n1: ──X───║───║──{label}─┤     \n         ╚═══╝"
```

Comment:
One _could_ do this for (partial) human readability and line length (black won't break these strings, afaik?)
Consider it a very weak suggestion :sweat_smile: 

```suggestion
            f"0: ──X──┤↗├──X──{label}─┤  <Z>\n"
            f"1: ──X───║───║──{label}─┤     \n"
            "         ╚═══╝"
```

Reply:
I did have it that way initially but black kept reformatting it :(

Reply:
Fair enough, then it's not supposed to be :D